code/WorkInProgress/TempEngine.dm

```diff
@@ -733,6 +818,10 @@ datum/pump_ui/circulator_ui
 				running = 1
 			SPAWN_DBG(0.5 SECONDS)
 				spam_limiter = 0
+		else if(warnings > WARNING_5MIN && !(src.status & (BROKEN | NOPOWER)))
+			// Allow for klaxon to trigger when off cooldown if updateicon() not called
+			if(!ON_COOLDOWN(src, "klaxon", 5 SECOND))
```

Comment:
5 seconds could get annoying, but i guess it's supposed to be?

Reply:
Seemed fine while I was capturing screenshots and running around the engine room but decided to try to do other things while alarm was going and started to go insane.

Variable pitch was also just problematic as low end of varying pitch would stretch clip out to approximately 5 seconds. 😞 

It is enough to be somewhat jarring and a more consistent pitch feels much cleaner.